poetry.lock

```diff
@@ -8,16 +8,17 @@ python-versions = "*"
 
 [[package]]
 name = "astroid"
-version = "2.5.1"
+version = "2.11.7"
```

Comment:
There are a lot of updates here to deps that aren't part of this. I think you're looking for the `no-update` flag to poetry, IIRC: https://stackoverflow.com/questions/58961497/how-to-update-poetrys-lock-file-without-upgrading-dependencies

Reply:
Looks like lots of upgrades to the lock file are back, right?

Reply:
I swear- ive never done anything but use --no-update

I promise... but ill go check.